### ISSUE TITLE
[CoE Starter Kit - QUESTION] DLP Policy Associated View [blank]

### ISSUE THREAD

**Issue by ClaudioRWS** [microsoft/coe-starter-kit#7062]
### Does this question already exist in our backlog?

- [X] I have checked and confirm this is a new question.

### What is your question?

Olá pessoal!?
![dlp](https://github.com/microsoft/coe-starter-kit/assets/94450591/eb1521b5-c0dd-430a-b943-47c284d7695c)

I noticed that the issues are related to DLP impact, but there is a view related to the environment, this one is showing blank,

Is this the behavior?

Is there a flow that lists DLPs by environment?

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

4.16

### What method are you using to get inventory and telemetry?

Cloud flows

**Comment by manuelap-msft** [comment]
That is expected behaviour. We only sync policies that are marked as "visible to the maker" via the Environment Request management tooling, so currently not all policies are synced. If you don't have any policies marked as shared via the Env Request tooling this would be empty here.

**Comment by ClaudioRWS** [comment]
@manuelap-msft  I understand, it would be interesting in the future, who knows, to query the environment and visualize the DLPs, for those who are managing many environments (and I mean many environments) it would be super precious information

tks

**Comment by Jenefer-Monroe** [comment]
I believe you are asking for a feature to track which DLPs impact an environment. Is that what you are asking for?

**Comment by ClaudioRWS** [comment]
@Jenefer-Monroe That's right, it's the need to know which DLPs exist in the environment and not know what the impact would be.

Example, I have 2023 environments and I need to inform the operations sector which DLPs are in environment X.

I believe this type of information within a table in the CoE would be very relevant

**Comment by Jenefer-Monroe** [comment]
Got it, thank you. Ill place on the feature backlog

**Comment by harini-2-y** [comment]
This issue is being closed as its scope is now included in a newly created consolidated enhancement request for centralized management of orphaned components (Apps, Flows, Connection References) in the CoE Starter Kit.

New Issue Reference: [[CoE Starter Kit - Feature]: https://github.com/https://github.com/microsoft/coe-starter-kit/issues/10339

Thank you for your contributions
